addon/components/lt-body.js

```diff
@@ -336,7 +335,12 @@ export default Component.extend({
       _scrollTo,
       scrollToRow,
       _scrollToRow
-    } = this.getProperties(['scrollTo', '_scrollTo', 'scrollToRow', '_scrollToRow']);
+    } = this.getProperties([
+      'scrollTo',
+      '_scrollTo',
+      'scrollToRow',
+      '_scrollToRow'
+    ]);
```

Comment:
For example, I wouldn't write these destructuring `getProperties` statements anymore. Instead I'd go with multiple `get` lines.

I'll change that, when I'm back in the office.